src/Microsoft.ML.TimeSeries/STL/LocalRegression.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML.Internal.CpuMath.Core;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// this class is used to maintain the neighbors of a given particular point.
+    /// </summary>
+    internal class LocalRegression
+    {
+        private const double NumericalThreshold = 1.0e-10;
+        private readonly IReadOnlyList<double> _x;
+        private readonly IReadOnlyList<double> _y;
+        private int _length;
+
+        /// <summary>
+        /// the model is learned by several iterations of local weighted regression.
+        /// </summary>
+        private PolynomialModel _model = null;
```

Comment:
The CI is failing, and among the errors it's that you can't initialize this member outside the constructor
`error MSML_NoInstanceInitializers: Member _model has a field initializer outside the constructor`

Reply:
Updated.

---
In reply to: [435637725](https://github.com/dotnet/machinelearning/pull/5202#discussion_r435637725) [](ancestors = 435637725)